src/clarity-angular/datagrid/datagrid.html

```diff
@@ -32,11 +32,11 @@
                       [ngForTemplate]="iterator.template"></template>
             <ng-content *ngIf="!iterator"></ng-content>
         </div>
-    </div>
 
-    <!-- Custom placeholder overrides the default empty one -->
-    <ng-content select="clr-dg-placeholder"></ng-content>
-    <clr-dg-placeholder *ngIf="!placeholder"></clr-dg-placeholder>
+        <!-- Custom placeholder overrides the default empty one -->
+        <ng-content select="clr-dg-placeholder"></ng-content>
+        <clr-dg-placeholder *ngIf="!placeholder"></clr-dg-placeholder>
+    </div>
```

Comment:
Once again, what's this doing here? This is a major change that completely impacts our column size calculator. Right now, I'm pretty sure it's actually breaking it.